DigitalLearningSolutions.Web/Startup.cs

```diff
@@ -93,8 +93,8 @@ public void Configure(IApplicationBuilder app, IMigrationRunner migrationRunner,
                 app.UseBrowserLink();
             }
 
-            app.UseExceptionHandler("/LearningPortal/Error");
-            app.UseStatusCodePagesWithReExecute("/LearningPortal/StatusCode/{0}");
+            app.UseExceptionHandler("/Frameworks/Error");
+            app.UseStatusCodePagesWithReExecute("/Frameworks/StatusCode/{0}");

```

Comment:
It's probably worth moving these two actions to their own controller, e.g. a LearningSolutionsController? It's probably also worth moving the AccessibilityHelp and Terms actions so that those footer links can be shared as well. What do you think? I'm happy to make that change if you prefer 🙂

Reply:
Good point - I hadn't tested the AccessibilityHelp and Terms links but they don't work from elsewhere. It might be safer for you to make that change if you're happy to?

Reply:
Sure, happy to. Once this has been merged I'll do that and open a PR 🙂